sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryUtils.java

```diff
@@ -679,6 +714,20 @@ private static Object convertAvroArray(
     return ret;
   }
 
+  private static Object convertAvroMap(
```

Comment:
nit: The name might be a little confusing, this isn't actually converting an Avro Map type. Something like `convertAvroRecordToMap` might reduce that risk.

Reply:
Agreed, that's a more accurate name and I have updated it to that.